.github/workflows/reusable-ws-e2e.yml

```diff
@@ -0,0 +1,52 @@
+name: E2E WebSocket Tests
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+on:
+  workflow_call:
+
+    inputs:
+      ee:
+        description: 'use the ee version of WebSockets'
+        required: false
+        default: false
+        type: boolean
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  e2e_ws:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    timeout-minutes: 80
+
+    permissions:
+      contents: read
+      packages: write
+      deployments: write
+      id-token: write
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # checkout with submodules if token is provided
+      - uses: actions/checkout@v3
+       # checkout without submodules
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup-project
+      - uses: ./.github/actions/setup-redis-cluster
+      - uses: mansagroup/nrwl-nx-action@v3
+        with:
+          targets: lint
+          projects: "@novu/ws"
+
+      - name: Install wait-on plugin
+        run: npm i -g wait-on
+
+        # Runs a single command using the runners shell
+      - name: Build WS
+        run: CI='' pnpm build:ws
+
+      - name: Run unit tests
+        env:
+          IN_MEMORY_CLUSTER_MODE_ENABLED: false
```

Comment:
Not needed to enable for now as this environment variable is not yet used in WS.